.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,6 @@
+eclipse.preferences.version=1
```

Comment:
Please do not commit files that are unnecessary (e.g., files generated by IDEs such as this one and all the files in .settings) to the functionality you are writing to add